pkg/odo/cli/service/service_catalog_backend.go

```diff
@@ -71,17 +70,6 @@ func (b *ServiceCatalogBackend) CompleteServiceCreate(o *CreateOptions, cmd *cob
 			o.ServiceName = o.ServiceType
 		}
 
-		// we convert the param list provided in the format of key=value list
-		// to a map
-		o.ParametersMap = make(map[string]string)
```

Comment:
why are we removing this from the service catalog version?

Reply:
its moved to top level so SC and OH services can both use it https://github.com/girishramnani/odo/blob/360e98774f5eed265869524281280f874f40b0ad/pkg/odo/cli/service/create.go#L103

Reply:
fixed